app/components/explore/side-bar.js

```diff
@@ -16,6 +16,8 @@ export default class SideBar extends Component {
   customEndDate = null;
   @tracked showFilters = false;
   isMapVisible = true;
+  lat = 27.00;
+  lng = 79.00;
```

Comment:
But this is static so what's the point

Reply:
it's just initial, means I will change it.